pop-api/integration-tests/src/local_fungibles.rs

```diff
@@ -327,26 +325,26 @@ fn transfer_works() {
 		// Asset does not exist.
 		assert_eq!(
 			decoded::<Error>(transfer(addr.clone(), 1, BOB, amount,)),
-			Module { index: 52, error: 3 },
+			Ok(Module { index: 52, error: 3 }),
```

Comment:
Feels weird that those errors are wrapped in an `Ok()`

Reply:
`Ok()` here means decoded successfully so I think it makes sense.